pkg/extflag/pathorcontent.go

```diff
@@ -0,0 +1,70 @@
+package extflag
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/pkg/errors"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+// PathOrContent is a flag type that defines two flags to fetch bytes. Either from file (*-file flag) of content (* flag).
+type PathOrContent struct {
+	flagName string
+
+	required bool
+
+	path    *string
+	content *string
+}
+
+type CmdClause interface {
+	Flag(name, help string) *kingpin.FlagClause
+}
+
+// RegisterPathOrContent registers PathOrContent flag in kingpinCmdClause.
+func RegisterPathOrContent(cmd CmdClause, flagName string, help string, required bool) *PathOrContent {
+	fileFlagName := fmt.Sprintf("%s-file", flagName)
+	contentFlagName := flagName
+
+	fileHelp := fmt.Sprintf("Path to %s", help)
```

Comment:
```suggestion
	fileHelp := fmt.Sprintf("Path to %s (has priority over the content flag)", help)
```
Might be worth to add the priority somewhere but not sure if this is the right way/place.

Reply:
I will add